internal/utils/misc.go

```diff
@@ -118,6 +120,9 @@ var (
 	DenoPathOverride string
 )
 
+//go:embed eszip/*
+var eszipEmbedDir embed.FS
+
```

Comment:
nitpick: can we move this to below line 84?

Reply:
fixed in 2189242